app.json

```diff
@@ -18,7 +18,9 @@
             "generator": "secret"
         }
     },
-    "buildpacks": {
-            "url": "heroku/python"
-    }
-}
+    "buildpacks": [
+      {
+        "url": "heroku/python"
+      }
+    ]
+  }
```

Comment:
> why is it required ?

Because that’s how the POSIX standard defines a line.
>"**3.206 Line:**
A sequence of zero or more non- <newline> characters plus a terminating <newline> character."